lib/nib/history.rb

```diff
@@ -0,0 +1,32 @@
+module Nib::History
+  IRBRC = <<~'IRB'.freeze
```

Comment:
Yeah, that's how you call `String` methods on a heredoc's. For example `<<~'IRB'.downcase ...`